examples/semantic_segmentation/semantic_segmentation/seed_test_suite.py

```diff
@@ -29,6 +32,27 @@
 from kolena.workflow.annotation import SegmentationMask
 
 
+def within_range(area: int, range: Tuple[int, int]) -> bool:
+    return range[0] <= area < range[1]
+
+
+def seed_stratified_test_cases(complete_test_case: TestCase, test_suite_name) -> List[TestCase]:
+    test_samples = complete_test_case.load_test_samples()
+    test_cases = []
+    for size_name, area_range in SIZE_MAPPING_IMAGES.items():
+        samples = []
+        for ts, gt in test_samples:
+            image_size = ts.metadata["width"] * ts.metadata["height"]
+            if within_range(image_size, area_range):
+                samples.append((ts, gt))
+
+        if len(samples) > 0:
+            test_cases.append(
+                TestCase(f"{size_name} x image :: {test_suite_name}", test_samples=samples, reset=True),
```

Comment:
can we drop `x`? I think it can simply be "small image", "medium image", and "large image".

Reply:
will do